integration-tests/tests/cargo_near_new.rs

```diff
@@ -154,7 +154,7 @@ fn run_cargo_near_new() -> color_eyre::Result<camino::Utf8PathBuf> {
         cargo_near::GlobalContext {
             config: Default::default(),
             offline: false,
-            teach_me: false,
+            verbosity: cargo_near::Verbosity::Interactive,
```

Comment:
and here also 
smthing similar
```rust
    let verbosity = match cli.teach_me{

       true => Verbosity::TeachMe,
      false => Verbosity::Interactive,
    };
 ```

Reply:
it's also worth looking at how it happens in deploy commands when delegating to `near_cli_rs` subcommand

Reply:
It's tests so probably fine